releases/release-5.3.0.md

```diff
@@ -0,0 +1,388 @@
+| :---------- | :----------- | :----------- | :----------- |
+| TiDB | [`prepared-plan-cache.capacity`](/tidb-configuration-file.md#capacity)  | Modified | Controls the number of cached statements. The default value is changed from `100` to `1000`.|
+| TiKV | [`storage.reserve-space`](/tikv-configuration-file.md#reserve-space) | Modified | Controls space reserved for disk protection when TiKV is started. Starting from v5.3.0, 80% of the reserved space is spared for operations and maintenance when disk space is insufficient, and the remaining 20% space stores temporary disk files. |
+| TiKV | memory-usage-limit | Modified  | This configuration item is new in TiDB v5.3.0 and its value is calculated based on storage.block-cache.capacity. |
+| PD | [`log.file.max-days`](/pd-configuration-file.md#max-days) | Modified | Controls the maximum number of days that logs are retained. The default value is changed from `1` to `0`. |
+| PD | [`log.file.max-backups`](/pd-configuration-file.md#max-backups) | Modified | Controls the maximum number of logs that are retained. The default value is changed from `7` to `0`.  |
+| PD | [`patrol-region-interval`](/pd-configuration-file.md#patrol-region-interval) | Modified | Controls the running frequency at which replicaChecker checks the health state of a Region. The smaller this value is, the faster replicaChecker runs. Normally, you do not need to adjust this parameter. The default value is changed from `100ms` to `10ms`. |
+| PD | [`max-snapshot-count`](/pd-configuration-file.md#max-snapshot-count) | Modified | Controls the maximum number of snapshots that a single store receives or sends at the same time. PD schedulers depend on this configuration to prevent the resources used for normal traffic from being preempted. The default value is changed from `3` to `64`. |
+| PD | [`max-pending-peer-count`](/pd-configuration-file.md#max-pending-peer-count) | Modified | Controls the maximum number of pending peers in a single store. PD schedulers depend on this configuration to prevent too many Regions with outdated logs from being generated on some nodes. The default value is changed from `16` to `64`. |
+
+### Others
+
+- Temporary tables:
+
+    - Regarding local temporary tables, if you have created local temporary tables on a cluster earlier than TiDB v5.3.0, these tables are actually ordinary tables, and handled as ordinary tables after the cluster is upgraded to TiDB v5.3.0 or later versions. Regarding global temporary tables, if you have created global temporary tables on a cluster in TiDB v5.3.0 or later versions, when the cluster is downgraded to a version earlier than v5.3.0, these tables are handled as ordinary tables and a data error occures.
+    - TiCDC and BR support [global temporary tables](/temporary-tables.md#global-temporary-tables) since v5.3.0. If you use TiCDC and BR in a version earlier than v5.3.0 to replicate global temporary tables to the downstream, a table definition error occurs.
```

Comment:
```suggestion
    - Since v5.3.0, TiCDC and BR support [global temporary tables](/temporary-tables.md#global-temporary-tables). If you use TiCDC and BR of a version earlier than v5.3.0 to replicate global temporary tables to the downstream, a table definition error occurs.
```